R/internal.R

```diff
@@ -2,3 +2,93 @@
     # works like message() but prepends a timestamp
     message(date(), ": ", ...)
 }
+
+.rgbaConversion <- function(color, alpha = 0.5){
```

Comment:
Although these are internal functions, but for the package maintainers benefit, can we add simple documentation for these functions about:
* Their functionality
* input
* output